packages/block-library/src/navigation/index.php

```diff
@@ -426,10 +426,25 @@ function render_block_core_navigation( $attributes, $content, $block ) {
 		wp_enqueue_script( 'wp-block-navigation-view' );
 	}
 
-	$should_load_modal_view_script = isset( $attributes['overlayMenu'] ) && 'never' !== $attributes['overlayMenu'];
-	if ( $should_load_modal_view_script ) {
-		wp_enqueue_script( 'wp-block-navigation-view-modal' );
-	}
+	// Register @alpine/focus plugin
+	wp_register_script(
+		'AlpineJS_focus',
+		'https://unpkg.com/@alpinejs/focus@3.10.3/dist/cdn.min.js',
+		array(),
+		"3.10.3",
```

Comment:
Not like super important for the experiment, but it looks like adding the version causes a 302 redirection in `unpkg.com`:

<img width="500" alt="Screenshot 2022-09-21 at 12 48 38" src="https://user-images.githubusercontent.com/3305402/191485713-ed0bf345-a8fb-4ad0-9698-29b6eaabaa3c.png">

```suggestion
		null,
```